projects/new-getting-started-docs-and-reference-application.md

```diff
@@ -0,0 +1,113 @@
+
+The first step is to create a "specification" for this application. This involves defining the type of application we will use and how it should function. An example can be found in the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository, specifically in the [demo-app-specification.md](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/main/getting-started-guides/demo-app-specification.md):
+
+```markdown
+# Specification for the Getting Started Guide demo applications
+
+## Application
+
+1. Must use port 8080
+
+2. User must be able to access the endpoint http://localhost:8080/fibonacci?n=[input], and endpoint should return the following JSON response:
+
+- For valid input, `{"n":5,"result":5}`
+- For invalid input, `{"message":"n must be 1 <= n <= 90."}`
+```
+
+This specification will outline an application that demonstrates a wide range of OpenTelemetry features. Unlike the [Demo](https://github.com/open-telemetry/opentelemetry-demo/), this will be a single, straightforward application implemented in all supported languages, with simple business logic (e.g., Fibonacci calculations or a "roll the dice" feature).
+
+In the second step, this specification should be expanded to include instructions for the "Instrumentation Journey." This will outline the abstract, language-agnostic steps an end user will follow to integrate the OpenTelemetry SDK into the application, add instrumentation, export telemetry, and more.
+
+Once the specification is available, each SIG that implements the API and SDK in their respective language can provide the application in both uninstrumented and instrumented forms as examples in their repository.
+
+Once at least one SIG provides this application, the documentation can be updated to include the new learning experience, utilizing this application. We will leverage existing tools to [extract code excerpts](https://github.com/open-telemetry/opentelemetry.io/tree/main/tools) from external repositories and import the code directly into the website.
+
+An example of how this could look can be seen in this preview of the OpenTelemetry Documentation:
+
+<https://deploy-preview-5376--opentelemetry.netlify.app/docs/learn/getting-started/>
+
+## Deliverables
+
+- A specification for the reference application
+- Implementations of the reference application across all supported languages
+- Prose detailing the new learning experience on the OpenTelemetry website
+
+## Staffing / Help Wanted
+
+This project is a collaboration between SIG Communications (Docs Contributors) and the SIGs responsible for implementing the API and SDK in their respective languages. No new SIGs need to be formed; however, we do need individuals to sign up and support this effort:
+
+- Staffing from SIG Communications
+
+  - [@svrnm](https://github.com/svrnm) (Project Lead)
+
+- Staffing from SIG Java
```

Comment:
Let's try to get a few more volunteers to build the prototypes for different languages.

This should be pretty straight forward and not very time consuming. 

Super high value-to-effort ratio!